### ISSUE TITLE
Прерывания звука дверей в Russian Hexen 4.2

### ISSUE THREAD

**Issue by Kenon2** [Russian-Doom/russian-doom#271]
Заметил такое: при открытии простых дверей (которые открываются вверх) звук начинает проигрываться и сразу прекращает.

**Comment by JNechaevsky** [comment]
Спасибо, удалось повторить. Очень интересная ситуация, нужно поразбираться в чём дело, т.к. чего-чего, а код дверей не менялся. В оригинальной версии для DOS и Chocolate Hexen этой проблемы нет.

**Comment by JNechaevsky** [comment]
Не совсем понятно в чём было дело, но исправилось само, предположительно с обновлением библиотеки SDL. Будет исправлено в следующем релизе, спасибо.

**Comment by JNechaevsky** [comment]
Теперь действительно исправлено. Спасибо ещё раз.

**Comment by JNechaevsky** [comment]
Нет, не всё так просто. Проблема происходит в анкап-режиме (иными словами, когда fps выше 35). Будем разбираться.
